study.md

```diff
@@ -19,38 +19,55 @@ Please write the code that would take the following integer `007` and return the
 string `"700"`
 
 ```ruby
-  # your answer here
+
+007.to_s.rjust(3, '0').reverse
```

Comment:
Thank you. Google search helped. 

Resource: 
https://cbabhusal.wordpress.com/2014/09/30/adding-precedingleading-zeros-in-integers-in-rubyonrails/

Used for testing:
https://repl.it/languages/ruby